media/main.js

```diff
@@ -43,7 +44,8 @@ function handleMessage(message) {
       webviewDebugLogger(
         `Received dailySummary: ${JSON.stringify(message.data)}`
       );
-      addDailySummary(message.data);
+      addJiraTicketsToDailySummary(message.data.jiraTickets);
```

Comment:
Should this be under Daily Summary?

Reply:
How is this a critical error?

Reply:
No but prevents debugging down the road.

Reply:
This is a non-essential feature and doesn't deserve so much attention. We're not even sure if we're gonna keep it/drastically change it